pykern/pkunit.py

```diff
@@ -551,18 +551,21 @@ def _failed_msg(process):
                 )
             return r + self._update_message
 
-        def _ndiff_config(epsilon, work_d):
+        def _ndiff_config(epsilon, abs_epsilon, work_d):
             return pykern.pkio.write_text(
-                work_d.join("ndiff_conf.txt"), f"* * abs={float(epsilon)}"
+                work_d.join("ndiff_conf.txt"),
+                f"* * abs={float(abs_epsilon)}"
```

Comment:
I think epsilon should be rel_epsilon. There now needs to be a test to be sure that one or the other is passed. I comment on this below.